packet_test.go

```diff
@@ -213,6 +215,18 @@ func TestRecvPacket(t *testing.T) {
 	}
 }
 
+func TestRecvPacketTooShort(t *testing.T) {
+	for _, packet := range [][]byte{
+		{0, 0, 0, 0},
+		{0, 0, 0, 3},
+		{0, 0, 0, 4, sshFxpInit},
+		{0, 0, 0, 200, sshFxpData, 1, 1, 1, 1, 1, 1},
+	} {
+		_, _, err := recvPacket(bytes.NewReader(packet), nil, 0)
+		assert.Equal(t, errShortPacket, err)
```

Comment:
You don’t need to bring in a library to write err == errShortRead.

Reply:
Ok, removed.